### ISSUE TITLE
global, N.H., and S.H. standard regions proposal

### ISSUE THREAD

**Issue by taylor13** [cf-convention/discuss#325]
**Proposer's name** Karl Taylor

**Date** 5 June 2024

This is a proposal to add three "regions" to the [list of "standard regions"](https://cfconventions.org/Data/cf-standard-names/docs/standardized-region-names.html), namely "global", "northern_hemisphere", and "southern_hemisphere".  These regions were excluded from the list because they are "regions that could be specified by coordinate ranges in CF (e.g. western hemisphere)". 

The specific use case motivating this proposal is that for CMIP7, there is a request to be able to use a single variable to report time-series of the N.H., S.H., and global mean concentrations of certain trace gases.  This could be done using the CF-recommended method by defining a latitude coordinate of length 3 with latitude coordinate bounds of [-90, 0], [-90, 90], and [0, 90] and coordinate values of, say, -45, 0, and 45.  I think, however, that it would be clearer to most users if we simply defined a region index dimension and used the ``coordinates`` attribute to point to a variable recording the names of the regions.   We can, of course, already do this legally within the convention constraints, but my understanding is that we could not identify the auxiliary coordinate variable using the ``standard_name="region"``, because currently northern_hemisphere, southern_hemisphere, and global are not included in the standard list.  By adding these names to the CF list of standard regions, we could use the standard_name, "region", in this use case, which I think is helpful.

**Comment by davidhassell** [comment]
Hi @taylor13  - this sounds sensible and I support it. Thanks.

**Comment by JonathanGregory** [comment]
Dear Karl @taylor13 

Yes, I think that's fair enough. I support the proposal.

By the way, I wonder whether `global` should have been `globe` and if so whether it's worth allowing `globe`. It's the only region name which isn't a noun.

Cheers

Jonathan

**Comment by taylor13** [comment]
``globe`` would certainly be more consistent.  Will changing it now cause any grief?  Do we already have aliases for regions, like we do for standard names?

**Comment by JonathanGregory** [comment]
I don't believe we have aliases, but I think we can define a new region that means the same as the existing one, and recommend the new one. J

**Comment by larsbarring** [comment]
@taylor13 I support the proposal to add "northern_hemisphere" and "southern_hemisphere" to the list of standardized regions.

With respect to "GLOBAL" I note that the header text to the [CF Standardized Region List](https://cfconventions.org/Data/standardized-region-list/standardized-region-list.current.html), now at version 4, states
>   Version 1 of this list was based on the NASA GCMD keyword list for locations which was valid on 12 December 2002. ...
> ... Our intention is to keep this list consistent with GCMD.

And the current [GCMD list of keywords](https://gcmd.earthdata.nasa.gov/static/kms/) provides a list of regions as [CSV](https://gcmd.earthdata.nasa.gov/kms/concepts/concept_scheme/locations/?format=csv&page_num=1&page_size=2000) that still includes "**GLOBAL**" (but not GLOBE), which then can be further divided into "GLOBAL LAND" and "GLOBAL OCEAN". It think that we should stick to the expressed intention. 

Furthermore, I would suggest that we whenever possible avoid introducing aliases as these introduce a source for mistakes and confusion, not least in relation to existing software.

**Comment by taylor13** [comment]
I find @larsbarring argument is compelling and agree we should retain "global" and not include "globe".  (We've inherited the inconsistency found in the GCMD list.)

**Comment by JonathanGregory** [comment]
Yes, that's fair enough. We can regard the inconsistency of `global` as an amusing feature of the CF convention.
